extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -172,3 +173,22 @@
 .i-amphtml-lbv[gallery-view] .amp-lbv-button-slide {
   display: block;
 }
+
```

Comment:
This is actually `ImageViewer` specific css. I added it here in order to avoid breaking functionality, but I'm open to suggestions if we want to add it to amp.css, or if there is a more appropriate place to put it. Basically this is duplicate with existing css in `amp-image-lightbox` and it should be refactored to be prefixed with `image-viewer`. I'm looking for feedback on where this css should go.

Reply:
We don't have a good way of sharing CSS between components. `amp.css` works but because `amp.css` endup in the core `v0,js`, we want to be mindful of its size.

Given the plan to deprecate the image-lightbox code later, duplication is fine here IMO.